### DISCUSSION TITLE
Can't it be used in STM32 Nucleo-144?

### DISCUSSION THREAD

**Discussion by JoungwookLee** [JAndrassy/ArduinoOTA#229] [Q&A]
I'm  "Nucleo F429ZI" user. 
If I select "Generic STM32F4 Series", I will see the virtual programmer in IDE Tools, but if I select "Nucleo-144", I will not see the virtual programmer.
Can't it be used in STM32 Nucleo-144?

![image](https://github.com/JAndrassy/ArduinoOTA/assets/151011069/8cefe03e-9054-4b7b-8f75-22aa10f856f2)

**Comment by JAndrassy** [comment]
what is "virtual programmer"?

**Reply by JoungwookLee** [reply]
If I edit programmers.txt, "Arduino OTA (...)" should appear in the Tools menu, but it does not appear when I select Nucleo-144.

**Comment by JAndrassy** [comment]
It can be this problem https://github.com/arduino/arduino-ide/issues/591

I see the programmer there
![image](https://github.com/JAndrassy/ArduinoOTA/assets/10706773/2ac4ae9e-b986-47df-be5c-adab62a3678d)
